clang/lib/CodeGen/Targets/AArch64.cpp

```diff
@@ -133,7 +133,7 @@ class AArch64TargetCodeGenInfo : public TargetCodeGenInfo {
       }
     }
     auto *Fn = cast<llvm::Function>(GV);
-    BPI.setFnAttributes(*Fn);
+    CGM.getTargetCodeGenInfo().setFnAttributes(BPI, *Fn);
```

Comment:
```suggestion
    setFnAttributes(BPI, *Fn);
```
We're inside AArch64TargetCodeGenInfo here, so I think this is just a member function?

Reply:
oh right.